test/e2e/framework/util.go

```diff
@@ -4396,7 +4396,7 @@ func OpenWebSocketForURL(url *url.URL, config *restclient.Config, protocols []st
 	if err != nil {
 		return nil, fmt.Errorf("failed to create tls config: %v", err)
 	}
-	if tlsConfig != nil {
+	if tlsConfig != nil || url.Scheme == "https" {
```

Comment:
url.Scheme without the tlsConfig is probably sufficient.

Reply:
I will change it

Reply:
Changed!